app/src/main/java/org/openobservatory/netprobe/utils/Alert.java

```diff
@@ -52,4 +52,57 @@ public void onClick(DialogInterface dialog, int id) {
 
                 }).show();
     }
+
+    public static String formatString(String text){
```

Comment:
@lorenzoPrimi have you checked whether this could be done using `Gson`? I'd rather merge a pull request containing non custom code for increased maintainability.